Makefile

```diff
@@ -38,10 +38,10 @@ pep8 :
 	flake8 examples/ ot/ test/
 
 test : FORCE pep8
-	python -m py.test -v test/
+	python -m py.test -v test/ --cov=ot --cov-report html:cov_html
 	
 pytest : FORCE 
-	python -m py.test -v test/	
+	python -m py.test -v test/ --cov=ot
```

Comment:
Under Debian/Ubuntu logilab-common install a useless executable named pytest. It's a well known bug buttakes time to be corrested. This line ensure that the proper py.test is executed.

https://github.com/pytest-dev/pytest/issues/1833